orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -248,17 +243,29 @@ def to(self, *args, **kwargs):
                 self._device = torch.device(device_str)
         return super(ORTModule, self).to(*args, **kwargs)
 
+    def eval(self: T) -> T:
+        self._is_training = False
+        self._original_module.eval()
+
+    def train(self: T, mode: bool = True) -> T:
+        self._is_training = mode
+        self._original_module.train(mode)
+        if self._is_training and self._device.type == 'cuda':
+            torch.cuda.empty_cache()
+
     def forward(self, *inputs, **kwargs):
         '''Forward pass starts here and continues at `_ORTModuleFunction.forward`
 
         ONNX model is exported the first time this method is executed.
-        Next, a full training graph is splitted in forward and backward graph which are used
-        to instantiate ONNX Runtime InferenceSession`s
+        Next, we build a full training graph with module_gradient_graph_builder. 
+        Finally, we instantiate the ONNX Runtime InferenceSession.
         '''
+        if not self._is_training:
```

Comment:
If we are going to use ORT for eval at a later point, may we add a TODO here?

Reply:
added.